M/mosquitto_client/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "mosquitto_client"
+version = v"2.0.15"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/eclipse/mosquitto.git", "dc75fec606d5c266cff5b3cf2259beee4523cda7")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/mosquitto/
+
+# Fix "cannot finding openssl" under Windows
+if [[ ${target} == *-w64* ]]; then
```

Comment:
Besides being hard to understand what it refers to, `*-w64*` matches also 32-bit Windows: https://docs.binarybuilder.org/stable/building/#Platforms
```suggestion
if [[ ${target} == x86_64-w64-mingw32 ]]; then
```